adapters/Pangle/PangleAdapter/Renderer/GADPangleRewardedRenderer.m

```diff
@@ -64,12 +64,12 @@ - (void)renderRewardedAdForAdConfiguration:
   }
 
   PAGRewardedRequest *request = [PAGRewardedRequest request];
-  request.adString = adConfiguration.bidResponse;
-  GADPangleRTBRewardedRenderer *__weak weakSelf = self;
+  request.adString = adConfiguration.bidResponse.length ? adConfiguration.bidResponse : nil;
```

Comment:
@wangyuanyuanyy 
Does Pangle SDK treat both `nil` and empty string `""` the same (as waterfall ad)?
If that is the case, we probably don't need this check.

Reply:
@wangyuanyuanyy Any update on this comment?

Reply:
Pangle SDK treat both nil and empty string "" is the same .I fixed it.